moshi/src/main/java/com/squareup/moshi/JsonAdapter.java

```diff
@@ -56,7 +57,16 @@ public final String toJson(T value) {
 
   /**
    * Encodes {@code value} as a Java model comprised of maps, lists, strings, numbers, booleans
-   * and nulls. The returned model is equivalent to calling {@link #toJson} to encode {@code value}
+   * and nulls.
+   * 
+   * <p>Values encoded using {@linkplain JsonWriter#value(double) value(double)} or {@linkplain
```

Comment:
I wanna draw attention to the arguments not the method names and blue links seem distracting. might even be better as `{@code }`?